test/bin/mirror_registry.sh

```diff
@@ -11,28 +11,21 @@ REGISTRY_HOST=${REGISTRY_HOST:-${MIRROR_REGISTRY_URL}}
 PULL_SECRET=${PULL_SECRET:-${HOME}/.pull-secret.json}
 LOCAL_REGISTRY_NAME="microshift-local-registry"
 
-retry_pull_image() {
-    for attempt in $(seq 3) ; do
-        if ! podman pull "$@" ; then
-            echo "WARNING: Failed to pull image, retry #${attempt}"
-        else
-            return 0
-        fi
-        sleep $(( "${attempt}" * 10 ))
-    done
-
-    echo "ERROR: Failed to pull image, quitting after 3 tries"
-    return 1
-}
-
 prereqs() {
     # Install packages if not yet available locally
     if ! rpm -q podman skopeo jq &>/dev/null ; then
         "${SCRIPTDIR}/../../scripts/dnf_retry.sh" "install" "podman skopeo jq"
     fi
-    podman stop "${LOCAL_REGISTRY_NAME}" || true
-    podman rm "${LOCAL_REGISTRY_NAME}" || true
-    retry_pull_image "${REGISTRY_IMAGE}"
+    # Pull the registry image locally
+    podman rm -f "${LOCAL_REGISTRY_NAME}" || true
```

Comment:
Do we really need to recreate the container every time this script runs? Can we make it a bit smarter: if the container exists, don't recreate?

Reply:
In such case we can leave it as it is.

Reply:
Let me revert my last commit then.